TYPO3.Neos/Resources/Public/JavaScript/Content/Navigate/NodeTree.js

```diff
@@ -226,6 +226,18 @@ define(
 						if (PublishableNodes.get('workspaceWidePublishableEntitySubjects').findBy('documentNodeContextPath', node.data.key)) {
 							$(nodeSpan).addClass('neos-dynatree-dirty');
 						}
+						$('a[title]', nodeSpan).tooltip({container: '#neos-application'});
+					},
+
+					onCustomRender: function(node) {
+						var nodeTypeLabel = I18n.translate(node.data.nodeTypeLabel),
+							tooltip = node.data.title;
+
+						if (nodeTypeLabel !== '' && tooltip.indexOf(nodeTypeLabel) === -1) {
```

Comment:
this currently breaks master since `tooltip` seems to be `null`.
`ContentModule-built.js?bust=f0da31ac2bd0:422 Uncaught TypeError: Cannot read property 'indexOf' of null`